de-lib/src/main/java/org/iplantc/de/apps/client/views/details/AppDetailsViewImpl.java

```diff
@@ -267,9 +270,13 @@ public void onClick(ClickEvent event) {
             public void execute() {
                 final Splittable appJson = AutoBeanCodex.encode(AutoBeanUtils.getAutoBean(app));
 
-                ReactAppDetailsView.renderToolDetails(toolsContainer.getId(),
-                                                      appearance,
-                                                      appJson);
+                ReactToolDetails.ToolDetailProps detailProps = new ReactToolDetails.ToolDetailProps();
+
+                detailProps.appearance = appearance;
+                detailProps.app = appJson;
+
+                ReactDOM.render(React.createElement(ReactToolDetails.ToolDetails, detailProps), DivElement.as(toolsContainer.getElement()));
```

Comment:
This is just what I was hoping for: we can choose any arbitrary element to render onto, instead of requiring an element's ID.